aws/resource_aws_iam_group_policy.go

```diff
@@ -105,17 +117,22 @@ func resourceAwsIamGroupPolicyRead(d *schema.ResourceData, meta interface{}) err
 		return err
 	}
 
-	d.Set("group", group)
-	d.Set("name", name)
-	d.Set("policy", policy)
-
-	return nil
+	if err := d.Set("policy", policy); err != nil {
+		return err
```

Comment:
If we are going to add error checking for these attributes, the errors should include context for operators and code maintainers to understand the issue, e.g.

```suggestion
		return fmt.Errorf("error setting policy: %s", err)
```

Reply:
Done.